### PR TITLE
Added inQuest details analyzer

### PR DESCRIPTION
**Pull request by i-karen**
Adding Analyzer for IntelOwl APIs
for issue #200

### CODE REVIEW

File: configuration/analyzer_config.json [hunk 1]
```diff
@@ -290,6 +290,18 @@
     "requires_configuration": true,
     "python_module": "haget_run"
   },
+  "Inquest_Details": {
+    "type": "observable",
+    "observable_supported": ["ip","url"],
+    "external_service": true,
+    "description": "Deep File Inspection - details",
+    "requires_configuration": true,
+    "python_module": "inquest_run",
```
Comment (Eshaan7) [inline configuration/analyzer_config.json:299]: `python_module` should be `inquest.InQuest` according to recent changes

**Comment by Eshaan7** [comment]
I changed the target branch. All PRs should be made against the `develop` branch as noted in the contributing guidelines. Please checkout to develop, pull and merge latest changes and update your PR by squashing all commits into one.

**Comment by Eshaan7** [comment]
Regarding the analyzer, we should have 3 configs/variants in `analyzer_config.json` i.e 

1. `InQuest_IOCdb` for `/api/iocdb/search` endpoint for `["ip", "url", "domain", "hash"]`.
2. `InQuest_REPdb` for `/api/repdb/search` endpoint for `["ip", "url", "domain", "hash"]`.
3. `InQuest_DFI` for `/api/dfi/search/ioc/{kind}` and `/api/dfi/search/hash/{source}` endpoints for types: `["ip", "url", "domain", "hash"]`. Here, we can have an _if_ condition, something like:

```python
if self.observable_classification == "hash":
    uri = f"/api/dfi/search/hash/md5?hash={self.observable_name}"
elif self.observable_classification in ["ip", "url", "domain"]:
    uri = f"/api/dfi/search/ioc/{self.observable_classification}?keyword={self.observable_name}"
else:
    raise AnalyzerRunException()
```

Only a single class can be used for all 3. The behaviour is switched on the basis of `inquest_analysis` param inside `additional_config_params`.

**Comment by mlodic** [comment]
Just another note: tests are missing. You can follow the doc and take example from other tests.

**Pull request closed without merging**